t/lib/OpenQA/Test/Database.pm

```diff
@@ -63,9 +63,9 @@ sub insert_fixtures ($self, $schema, $fixtures_glob = '*.pl') {
     my %ids;
     foreach my $fixture (glob "$fixtures_glob") {
 
-        my $info = eval path($fixture)->slurp;    ## no critic
```

Comment:
That's a string eval and needs to stay.

Reply:
Do you have an idea how we could get rid of the `if $@` in the next line then? Something like combining eval+try together?

Reply:
Two ideas:

```
chdir $cwd, croak "Could not insert fixture $fixture: $@" if ${@};   # ;-)
```

```
use Englsh;

chdir $cwd, croak "Could not insert fixture $fixture: $EVAL_ERROR" if $EVAL_ERROR;
```

Reply:
Hm. How about a comment `# nostyle:bare-eval` and in our style check we only complain about `if $@` with a negative look-ahead for that special comment?

Reply:
I would be ok with that.
I just thought that using `$EVAL_ERROR` would be a form of "I know what I'm doing, I cannot use try/catch here".

Reply:
yeah, I guess that works. Updated